avm/res/api-management/service/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -182,7 +182,7 @@ module testDeployment '../../../main.bicep' = [
         }
       ]
       managedIdentities: {
-        systemAssigned: true
+        systemAssigned: false
         userAssignedResourceIds: [
           nestedDependencies.outputs.managedIdentityResourceId
         ]
```

Comment:
Required for WAF-alignment? Like in general?

Reply:
Not requred for WAF. WAF requires that at least 1 MI is assigned to the APIM. I just wanted to have a different test case for my new MI logic than what is in the max deployment test case.

So:
max deployment = deploy both system assigned and user assigned 
waf deployment = deploy just user assigned

Reply:
Hey @tony-box,
thanks for elaborating.